src/components/Footer.tsx

```diff
@@ -34,21 +34,51 @@ export default async function Footer() {
     },
   ];
 
+  const socialMediaLinks = [
+    {
+      url: "https://www.instagram.com/project.protocol",
+      icon: "instagram",
+    },
+    {
+      url: "https://www.facebook.com/projectprotocol/",
+      icon: "facebook",
+    },
+    {
+      url: "https://www.tiktok.com/@projectprotocol",
+      icon: "tiktok",
+    },
+    {
+      url: "https://www.linkedin.com/company/projectprotocol/",
+      icon: "linkedin",
+    },
+  ];
+
   return (
-    <div
-      className="bg-dark text-center mt-auto py-md-4 pt-4"
-      style={{ paddingBottom: "100px" }}
-    >
-      {links.map(({ label, url, target }) => (
-        <Link
-          key={`footer-link-${label}`}
-          className="mx-3 my-3 d-block d-md-inline text-center link-white link-underline-opacity-0"
-          href={url}
-          target={target || "_self"}
-        >
-          {label}
-        </Link>
-      ))}
-    </div>
+    <>
+      <div className="bg-dark text-center mt-auto py-md-4 pt-4 pb-5">
+        {links.map(({ label, url, target }) => (
+          <Link
+            key={`footer-link-${label}`}
+            className="mx-3 my-3 d-block d-md-inline text-center link-white link-underline-opacity-0"
+            href={url}
+            target={target || "_self"}
+          >
+            {label}
+          </Link>
+        ))}
+        <span className="d-flex flex-wrap justify-content-center">
+          {socialMediaLinks.map(({ url, icon }) => (
+            <a
+              key={`social-link-${icon}`}
+              href={url}
+              target="_blank"
+              className="mx-3 my-3 d-block d-md-inline text-center link-white link-underline-opacity-0"
+            >
+              <i className={`bi bi-${icon} h1`}></i>
```

Comment:
```suggestion
              <i className={`bi bi-${icon}  fs-2`}></i>
```
For icons, the `font-size` class is standard for [sizing](https://icons.getbootstrap.com/#icon-font), so maybe something like this.